msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -8,15 +8,34 @@ ms.localizationpriority: medium
 keywords: teams apps meetings user participant role api  
 ---
 
-# Apps for Teams meetings
+# Apps for Teams meetings and calls
 
 Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting app can deliver a user experience for each stage of the meeting lifecycle. Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the attendee's status.
 
-The users can access apps during meetings using the tab gallery from their calendar, such as:
+> [!Note]
+>
+> Apps for instant meetings, one-on-one and group calls are currently available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
 
-* Pre-stage a Kanban board.
-* Launch an in-meeting actionable dialog.
-* Create a post-meeting survey.
+Teams supports access to apps during meeting for the following meeting types:
+
+* [**Scheduled meetings**](https://support.microsoft.com/en-us/office/schedule-a-meeting-in-teams-943507a9-8583-4c58-b5d2-8ec8265e04e5#ID0EFBD=Desktop): Meetings scheduled through Teams calendar.
+* [**One-on-one calls**](https://support.microsoft.com/en-us/office/start-a-call-from-a-chat-in-teams-f5138c9d-df4c-43d8-9cf6-53400c1a7798): Calls initiated in one-on-one chat.
+* [**Group calls**](https://support.microsoft.com/en-us/office/start-a-call-from-a-chat-in-teams-f5138c9d-df4c-43d8-9cf6-53400c1a7798): Calls initiated in group chat.
+* [**Instant meetings**](https://support.microsoft.com/en-us/office/start-an-instant-meeting-in-teams-ff95e53f-8231-4739-87fa-00b9723f4ef5): Meetings initiated through **Meet now** button in Teams calendar.
+
+Users can add apps to the meeting using the **+** option from their Teams meeting window.
+
+:::image type="content" source="../assets/images/apps-in-meetings/add-app.png" alt-text="Add an app in meeting" border="true":::
+
+Visit the [Teams store](https://go.microsoft.com/fwlink/p/?LinkID=2183121&clcid=0x4009) and explore apps designed specifically for meetings.
+
+> [!Note]
+>
+> * Add apps option is not available in Teams mobile client. But mobile client does support to view the app, [meeting SidePanel](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#meeting-sidepanel), and [share entire app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) when user attends a meeting through Teams mobile client.
```

Comment:
Let's standardize on how we refer to this concept. Right now it's all over the place-- I see us use "meeting sidepanel", "meeting side panel", "meetingSidePanel", "meeting SidePanel", "app side panel"